app.py

```diff
@@ -13,10 +13,25 @@
 # Create App and stacks
 app = core.App()
 
+artifact_bucket = app.node.try_get_context("ArtifactBucket")
+
 # Create the pipeline stack
-PipelineStack(app, "drift-pipeline")
+synth = core.DefaultStackSynthesizer()
+if artifact_bucket is not None:
+    synth = core.DefaultStackSynthesizer(
+        file_assets_bucket_name=artifact_bucket,
```

Comment:
Could we also pass in `bucket_prefix` so that I can provide `drit-pipeline` in this case?

Reply:
Done!